### ISSUE TITLE
Getting linking issues after building Pillow on Xcode 12.x

### ISSUE THREAD

**Issue by Cheaterman** [kivy/kivy-ios#585]
I'm on OSX 10.15.7 BTW, not 11.

Another thing, I thought it could be because of Python 3.9, so I downgraded to 3.8, but I see from your CI specs Mirko that you're using 3.9, so that's also a dead end - I'll go back to 3.9 myself. In the meantime, logs:
```
ld: warning: could not create compact unwind for _ffi_call_unix64: does not use RBP or RSP based frame
Undefined symbols for architecture x86_64:
  "_cmsCloseProfile", referenced from:
      _cms_profile_dealloc in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsCreateLab2Profile", referenced from:
      _createProfile in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsCreateProofingTransform", referenced from:
      _buildProofTransform in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsCreateTransform", referenced from:
      _buildTransform in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsCreateXYZProfile", referenced from:
      _createProfile in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsCreate_sRGBProfile", referenced from:
      _createProfile in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsDeleteTransform", referenced from:
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_transform_dealloc in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsDoTransform", referenced from:
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_transform_apply in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetColorSpace", referenced from:
      _cms_profile_getattr_color_space in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_xcolor_space in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetDeviceClass", referenced from:
      _cms_profile_getattr_device_class in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetEncodedICCversion", referenced from:
      _cms_profile_getattr_icc_version in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderAttributes", referenced from:
      _cms_profile_getattr_attributes in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderCreationDateTime", referenced from:
      _cms_profile_getattr_creation_date in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderFlags", referenced from:
      _cms_profile_getattr_header_flags in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderManufacturer", referenced from:
      _cms_profile_getattr_header_manufacturer in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderModel", referenced from:
      _cms_profile_getattr_header_model in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderProfileID", referenced from:
      _cms_profile_getattr_profile_id in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderRenderingIntent", referenced from:
      _cms_profile_getattr_rendering_intent in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetPCS", referenced from:
      _cms_profile_getattr_pcs in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_connection_space in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetProfileInfoASCII", referenced from:
      _cms_profile_getattr_product_desc in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_product_description in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_product_manufacturer in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_product_model in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_product_copyright in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetProfileVersion", referenced from:
      _cms_profile_getattr_version in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetSupportedIntents", referenced from:
      __is_intent_supported in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetTransformInputFormat", referenced from:
      _cms_transform_apply in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetTransformOutputFormat", referenced from:
      _cms_transform_apply in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsIsCLUT", referenced from:
      __is_intent_supported in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsIsIntentSupported", referenced from:
      _cms_profile_is_intent_supported in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      __is_intent_supported in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsIsMatrixShaper", referenced from:
      _cms_profile_getattr_is_matrix_shaper in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      ...
  "_cmsIsTag", referenced from:
      _cms_profile_getattr_technology in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_colorimetric_intent in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_perceptual_rendering_intent_gamut in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_saturation_rendering_intent_gamut in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      ...
  "_cmsMLUgetWide", referenced from:
      __profile_read_mlu in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsNamedColorCount", referenced from:
      __profile_read_named_color_list in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsNamedColorInfo", referenced from:
      __profile_read_named_color_list in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsOpenProfileFromFile", referenced from:
      _cms_profile_open in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsOpenProfileFromMem", referenced from:
      _cms_profile_fromstring in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsReadTag", referenced from:
      _cms_profile_getattr_technology in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_colorimetric_intent in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_perceptual_rendering_intent_gamut in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_saturation_rendering_intent_gamut in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      ...
  "_cmsSaveProfileToMem", referenced from:
      _cms_profile_tobytes in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsTempFromWhitePoint", referenced from:
      _cms_profile_getattr_media_white_point_temperature in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsWhitePointFromTemp", referenced from:
      _createProfile in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsXYZ2xyY", referenced from:
      _cms_profile_getattr_red_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_media_white_point_temperature in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      ...
  "_iconv", referenced from:
      _qr_code_data_list_extract_text in libzbar.a(libzbar_la-qrdectxt.o)
     (maybe you meant: _SDL_iconv_close, _SDL_iconv , _SDL_iconv_open , _SDL_iconv_string )
  "_iconv_close", referenced from:
      _qr_code_data_list_extract_text in libzbar.a(libzbar_la-qrdectxt.o)
     (maybe you meant: _SDL_iconv_close)
  "_iconv_open", referenced from:
      _qr_code_data_list_extract_text in libzbar.a(libzbar_la-qrdectxt.o)
     (maybe you meant: _SDL_iconv_open)
  "_opj_create_compress", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_create_decompress", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_decode_tile_data", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_destroy_codec", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_end_compress", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_end_decompress", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_image_create", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_image_destroy", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_read_header", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_read_tile_header", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_set_default_decoder_parameters", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_set_default_encoder_parameters", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_set_error_handler", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_set_info_handler", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_set_warning_handler", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_setup_decoder", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_setup_encoder", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_start_compress", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_create", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_destroy", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_read_function", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_seek_function", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_skip_function", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_user_data", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_user_data_length", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_write_function", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_version", referenced from:
      _ImagingJpeg2KVersion in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_write_tile", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

_Originally posted by @Cheaterman in https://github.com/kivy/kivy-ios/pull/579#issuecomment-760466311_

**Comment by misl6** [comment]
I will check it ASAP, you also had the chance to try a build for arm64 instead of the x86_64 simulator one?

**Comment by Cheaterman** [comment]
I didn't give it a try yet TBH, I assumed it wouldn't work - I'll still try tomorrow. Also, I'm using 3.8 from homebrew, which had me add some LDFLAGS and other env vars in .bash_profile to get 3.8 to work (and to be in PATH), I don't know if that's any different from your own setup?

**Comment by misl6** [comment]
Ok, I think I got it.

I'm able to compile and link it properly cause I don't have some dependencies on my system which Pillow is looking at during config.

Looking at your logs seems that  Pillow it's being built with `openjpg` and `littlecms` support.

Not a proper and definitive solution, but just as a workaround, can you please edit the following line by adding `--disable-lcms` and `--disable-jpeg2000` flags ?

https://github.com/kivy/kivy-ios/blob/2ba28a546db011ef380c1f0c05cd00c2eea26acf/kivy_ios/recipes/pillow/__init__.py#L37

Then, if it's working, maybe We need to properly target this specific issue via a patch in `1.3.0` during a version upgrade?

**Comment by Cheaterman** [comment]
Thanks, that sounds like a decent workaround, given I don't actually need these features, I'll give that a try and keep you in touch!

**Comment by Cheaterman** [comment]
@misl6 Thank you so much again Mirko, it definitely helped!!! I had similar issues with iconv, because of zbar (not pillow), I solved them by adding libiconv.tbd to the linking phase of the project, could you please tell me (if you know) how to do that in the recipe (even though it's slightly offtopic, it still seems somewhat relevant) ?

**Comment by rajathithan** [comment]
Can someone please help me on this issue , I get the below error when i compile the code in Xcode 12, I am not able to upgrade to pillow 8.2.0 version, it allows me to install only the 6.2.0 version and I am always ending up with the below error 

`ld: warning: could not create compact unwind for _ffi_call_unix64: does not use RBP or RSP based frame
Undefined symbols for architecture x86_64:
  "_cmsCloseProfile", referenced from:
      _cms_profile_dealloc in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsCreateLab2Profile", referenced from:
      _createProfile in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsCreateProofingTransform", referenced from:
      _buildProofTransform in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsCreateTransform", referenced from:
      _buildTransform in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsCreateXYZProfile", referenced from:
      _createProfile in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsCreate_sRGBProfile", referenced from:
      _createProfile in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsDeleteTransform", referenced from:
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_transform_dealloc in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsDoTransform", referenced from:
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_transform_apply in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetColorSpace", referenced from:
      _cms_profile_getattr_color_space in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_xcolor_space in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetDeviceClass", referenced from:
      _cms_profile_getattr_device_class in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetEncodedICCversion", referenced from:
      _cms_profile_getattr_icc_version in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderAttributes", referenced from:
      _cms_profile_getattr_attributes in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderCreationDateTime", referenced from:
      _cms_profile_getattr_creation_date in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderFlags", referenced from:
      _cms_profile_getattr_header_flags in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderManufacturer", referenced from:
      _cms_profile_getattr_header_manufacturer in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderModel", referenced from:
      _cms_profile_getattr_header_model in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderProfileID", referenced from:
      _cms_profile_getattr_profile_id in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetHeaderRenderingIntent", referenced from:
      _cms_profile_getattr_rendering_intent in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetPCS", referenced from:
      _cms_profile_getattr_pcs in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_connection_space in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetProfileInfoASCII", referenced from:
      _cms_profile_getattr_product_desc in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_product_description in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_product_manufacturer in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_product_model in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_product_copyright in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetProfileVersion", referenced from:
      _cms_profile_getattr_version in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetSupportedIntents", referenced from:
      __is_intent_supported in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetTransformInputFormat", referenced from:
      _cms_transform_apply in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsGetTransformOutputFormat", referenced from:
      _cms_transform_apply in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsIsCLUT", referenced from:
      __is_intent_supported in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsIsIntentSupported", referenced from:
      _cms_profile_is_intent_supported in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      __is_intent_supported in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsIsMatrixShaper", referenced from:
      _cms_profile_getattr_is_matrix_shaper in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      ...
  "_cmsIsTag", referenced from:
      _cms_profile_getattr_technology in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_colorimetric_intent in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_perceptual_rendering_intent_gamut in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_saturation_rendering_intent_gamut in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      ...
  "_cmsMLUgetWide", referenced from:
      __profile_read_mlu in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsNamedColorCount", referenced from:
      __profile_read_named_color_list in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsNamedColorInfo", referenced from:
      __profile_read_named_color_list in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsOpenProfileFromFile", referenced from:
      _cms_profile_open in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsOpenProfileFromMem", referenced from:
      _cms_profile_fromstring in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsReadTag", referenced from:
      _cms_profile_getattr_technology in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_colorimetric_intent in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_perceptual_rendering_intent_gamut in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_saturation_rendering_intent_gamut in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      ...
  "_cmsSaveProfileToMem", referenced from:
      _cms_profile_tobytes in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsTempFromWhitePoint", referenced from:
      _cms_profile_getattr_media_white_point_temperature in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsWhitePointFromTemp", referenced from:
      _createProfile in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
  "_cmsXYZ2xyY", referenced from:
      _cms_profile_getattr_red_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_colorant in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_red_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_green_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_blue_primary in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      _cms_profile_getattr_media_white_point_temperature in libpillow.a(_imagingcms.cpython-38-darwin.so.o)
      ...
  "_opj_create_compress", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_create_decompress", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_decode_tile_data", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_destroy_codec", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_end_compress", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_end_decompress", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_image_create", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_image_destroy", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_read_header", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_read_tile_header", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_set_default_decoder_parameters", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_set_default_encoder_parameters", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_set_error_handler", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_set_info_handler", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_set_warning_handler", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_setup_decoder", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_setup_encoder", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_start_compress", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_create", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_destroy", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_read_function", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_seek_function", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_skip_function", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_user_data", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_user_data_length", referenced from:
      _ImagingJpeg2KDecode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_stream_set_write_function", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_version", referenced from:
      _ImagingJpeg2KVersion in libpillow.a(_imaging.cpython-38-darwin.so.o)
  "_opj_write_tile", referenced from:
      _ImagingJpeg2KEncode in libpillow.a(_imaging.cpython-38-darwin.so.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
`

**Comment by misl6** [comment]
Fixed via #606
